openapi.yaml

```
@@ -0,0 +1,1367 @@
+          required: true
+        - name: "label_id"
+          in: "path"
+          description: "The part to lookup e.g. 433"
+          schema:
+            type: "string"
+          required: true
+      tags:
+        - "layer"
+      summary: "Retrieve internal citations for a part"
+      description: "Retrieve internal citations for a part"
+      responses:
+        "200":
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/InternalCitationsLayer"
+        "400":
+          description: "Invalid input"
+  /layer/paragraph-markers/cfr/{version_id}/{label_id}:
+    get:
+      parameters:
+        - name: "version_id"
+          in: "path"
+          description: "The version to lookup e.g. 2020-28567"
+          schema:
+            type: "string"
+          required: true
+        - name: "label_id"
+          in: "path"
+          description: "The part to lookup e.g. 433"
+          schema:
+            type: "string"
+          required: true
+      tags:
+        - "layer"
+      summary: "Retrieve paragraph markers for a part"
+      description: "Retrieve paragraph markers for a part"
+      responses:
+        "200":
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ParagraphMarkersLayer"
+        "400":
+          description: "Invalid input"
+  /layer/analyses/cfr/{version_id}/{label_id}:
+    get:
+      parameters:
+        - name: "version_id"
+          in: "path"
+          description: "The version to lookup e.g. 2020-28567"
+          schema:
+            type: "string"
+          required: true
+        - name: "label_id"
+          in: "path"
+          description: "The part to lookup e.g. 433"
+          schema:
+            type: "string"
+          required: true
+      tags:
+        - "layer"
+      summary: "Retrieve analyses for a part"
+      description: "Retrieve analyses for a part"
+      responses:
+        "200":
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                type: "object"
+        "400":
+          description: "Invalid input"
+  /notice:
+      get:
+        tags:
+          - "notice"
+        summary: "Return an array of Notices"
+        description: "Return an array of Notices"
+        responses:
+          "200":
+            description: "Successful operation"
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/NoticeList"
+          "400":
+            description: "Invalid value"
+  /notice/{docnum}:
+    get:
+      parameters:
+        - name: "docnum"
+          in: "path"
+          summary: "The document number for the notice"
+          description: "The document number for the notice"
```

apparently summary isn't a think for a parameter
```suggestion
          description: "The document number for the notice"
```